src/main/java/de/tum/in/www1/exerciseapp/domain/MultipleChoiceSubmittedAnswer.java

```diff
@@ -90,6 +90,25 @@ public void checkForDeletedAnswerOptions(MultipleChoiceQuestion question) {
         }
     }
 
+    /**
+     * Delete all references to question and answers if the question was changed
+     *
+     * @param quizExercise the changed quizExercise-object
+     */
+    public void updateForDeletedReferences (QuizExercise quizExercise) {
```

Comment:
same comment as above, please use a different method name of you actually delete something --> If I read "update", I would assume that nothing is deleted